.github/workflows/ci.yml

```diff
@@ -0,0 +1,78 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        rust:
+          - stable
+          - beta
+          - nightly
+
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Install toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{ matrix.rust }}
+          profile: minimal
+          override: true
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get install libssl-dev
+
+      - name: Check default-features
+        run: cargo check
+
+      - name: Check tls
+        run: cargo check --features tls
+
+      - name: Test
+        run: cargo test
```

Comment:
Was it meant to be `cargo test --release`?

Reply:
oh yeah, it's with --release in the travis config. It's originated from the workflow I created for async-tungstenite.